Gemfile

```diff
@@ -4,6 +4,9 @@ gemspec
 group :test do
   gem 'sinatra', '~> 1.4'
   gem 'rspec',   '~> 3.0'
+  if RUBY_VERSION =~ /^1\.9\.3.*/
+    gem 'public_suffix', '<= 1.4.6'
```

Comment:
Actually it was a guard for ensuring that 'webmock' would not bring in 'addressable' > 2.4.0 
https://rubygems.org/gems/webmock/versions/1.24.6
https://rubygems.org/gems/addressable/versions/2.4.0
https://rubygems.org/gems/addressable/versions/2.5.0 (brings in public_suffix that does not support Ruby 1.9.3)

You are right my guard does not sense, since the version of addressable (2.4.0) does not use public_suffix. So the guard should of cause be on addressable.

I'll update the PR